### PR TITLE
Fix ReSpec warnings

### PR DESCRIPTION
**Pull request by anssiko**
ReSpec now surfaces the warnings and errors more prominently (thanks @marcoscaceres!), so I started looking at fixing those for this spec.

I was able to get rid of the warnings, together with changes in #520 those are gone.

However, fixing errors requires more substantive change, opened #542 for that.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -602,7 +602,7 @@ <h3 id="installation-sec">
         <li>Otherwise, return <code>false</code>.
         </li>
       </ol>
-      <div class="issue" title="🐒 Monkey patch">
+      <div class="issue" title="&#x1f412; Monkey patch">
```
Comment (kenchris) [inline index.html]: You changed that in other patch... better remove from that commit?

Comment (anssiko) [inline index.html]: TL;DR: TextMate does not like unicode monkey.

**Comment by marcoscaceres** [comment]
Fixed the monkey manually :) Thanks again, @anssiko! 

@kenchris, please don't forget that you can edit nits directly via Github. Saves us a bit of back/forth.

**Comment by anssiko** [comment]
Thanks!

(Every time I touch this spec, the monkey keeps reminding me I should move on from TextMate. But I'm hooked.)

**Comment by marcoscaceres** [comment]
@anssiko, I know and understand. Don't worry :) I got my own software-drug problems (e.g., AirMail... it's a pile a garbage, but I. Can't. Break. Free.... an ReSpec... I. Love. Hacking. On. It. Send! Help!).

**Pull request merged**